superset/reports/commands/alert.py

```diff
@@ -166,6 +166,9 @@ def _execute_query(self) -> pd.DataFrame:
             logger.warning("A timeout occurred while executing the alert query: %s", ex)
             raise AlertQueryTimeout() from ex
         except Exception as ex:
+            logger.warning(
+                "A general exception occurred while executing the alert query: %s", ex
+            )
             raise AlertQueryError(message=str(ex)) from ex
```

Comment:
does this exception already log this error?

Reply:
Yes but i'm curious to see what this error is on it's face because it could be empty.